test/tools/unified-spec-runner/entities.ts

```diff
@@ -212,7 +212,7 @@ export class UnifiedMongoClient extends MongoClient {
       ...getEnvironmentalOptions(),
       ...(description.serverApi ? { serverApi: description.serverApi } : {}),
       // TODO(NODE-5785): We need to increase the truncation length because signature.hash is a Buffer making hellos too long
-      mongodbLogMaxDocumentLength: 1250
+      mongodbLogMaxDocumentLength: 1500
```

Comment:
Without this change, one logging test fails.  

The length of the full log message without compression comes out to `1233`, locally.  With compression enabled, `compression: ['<>']` makes the full stringified log message >1250 characters, so the log message is truncated.  This breaks the UTR, which, to assert on the log message, uses JSON.parse() on the log message (we've stringified an unparsable log message).

I just bumped the max length high enough I don't think we'll have any more issues going forward.